packages/astro/src/integrations/hooks.ts

```diff
@@ -609,6 +602,7 @@ export async function runHookBuildGenerated({
 type RunHookBuildDone = {
 	settings: AstroSettings;
 	pages: string[];
+	// TODO: remove in Astro 6
```

Comment:
We forgot to add this todo while doing https://docs.astro.build/en/guides/upgrade-to/v5/#deprecated-routes-on-astrobuilddone-hook-integration-api